package.json

```diff
@@ -1,63 +1,94 @@
 {
   "name": "videojs-playlist",
-  "version": "2.0.0",
   "description": "Playlist plugin for videojs",
-  "main": "index.js",
-  "scripts": {
-    "build": "npm run build-dist",
-    "watch": "npm run watch-dist & npm run watch-tests & wait",
-    "buildall": "npm run build-dist && npm run build-tests",
-    "build-dist": "mkdir -p dist/ && browserify . -s videojs-playlist -o dist/bundle.js && browserify . -s videojs-playlist -p deumdify > dist/global.js",
-    "watch-dist": "mkdir -p dist/ && watchify . -s videojs-playlist -v -d -o dist/bundle.js",
-    "build-tests": "mkdir -p dist/ && browserify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-playlist -d -o dist/tests.js",
-    "watch-tests": "mkdir -p dist/ && watchify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-playlist -v -d -o dist/tests.js",
-    "prepublish": "npm run build",
-    "test": "jshint src/**.js test/**.js index.js && npm run build-tests && karma start test/karma.conf.js --single-run",
-    "test-watch": "npm run watch-tests & karma start test/karma.conf.js --auto-watch & wait",
-    "jshint": "jshint src/**.js test/**.js"
-  },
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "version": "2.0.0",
```

Comment:
Hmm, yeah, I guess that's a feature of npm's package.json editing? It's likely because it's parsing the package.json as JSON then merging the new properties and calling `JSON.stringify` on the result.